SAS/Jonckheere-Terpstra Test.qmd

```diff
@@ -0,0 +1,106 @@
+------------------------------------------------------------------------
+
+title: "SAS Jonckheere-Terpstra Test"
+
+------------------------------------------------------------------------
+
+## Background
+
+The Jonckheere-Terpstra (JT) test is a nonparametric method designed to detect ordered differences across categories. It offers an advantageous alternative to more general tests for class differences, such as the Kruskal-Wallis test, particularly when the analysis is conducted using the WILCOXON option within the NPAR1WAY procedure. $^{[1]}$
+
+The JT test is particularly well-suited for dose-response or trend analysis with ordered categorical data, where the objective is to ascertain whether an increment in dosage leads to a corresponding escalation or reduction in the response variable.$^{[2]}$$^{[5]}$ Unlike other statistical evaluations that might focus on identifying isolated differences between groups, this test is specifically tailored to uncover an overarching trend within the data.
+
+## SAS Procedure
+
+To request Jonckheere-Terpstra test, specify the **JT** option in the Table statement like below:
+
+Preq; table Var1 \* Var2 / **JT** ; Quit;
+
+The JT option in the TABLES statement provides the Jonckheere-Terpstra test.
+
+PROC FREQ also provides exact p-values for the Jonckheere-Terpstra test. You can request the exact test by specifying the **JT** option in the EXACT statement.$^{[3]}$
+
+## Data used
+
+``` sas
+data ICU_Stay;
+    input ICU $ Stay;
+    label Stay = 'Length of Stay in Days';
+    datalines;
+Cardiothoracic 7
+Medical 4
+Cardiothoracic 1
+Medical 7
+Cardiothoracic 2
+Medical 16
+Cardiothoracic 6
+Medical 11
+Cardiothoracic 11
+Medical 21
+Cardiothoracic 8
+Neurosurgical 20
+Neurosurgical 25
+Neurosurgical 13
+Neurosurgical 9
+Neurosurgical 14
+Neurosurgical 11
+;
+run;
+proc sort data=ICU_Stay;
+    by ICU Stay;
+run;
+```
+
+``` sas
+data contin;
+input group subject response;
+cards;
+0 01 27
+0 02 28
+0 03 27
+0 04 31
+0 05 34
+0 06 32
+1 01 31
+1 02 35
+1 03 34
+1 04 32
+1 05 31
+1 06 33
+2 01 32
+2 02 33
+2 03 30
+2 04 34
+2 05 37
+2 06 36
+3 01 40
+3 02 39
+3 03 41
+3 04 38
+3 05 42
+3 06 43
+;
+run;
+```
+
+## Example Code using
+
+proc freq data=ICU_Stay; table ICU\*Stay / **JT** nopercent nocol norow; run;
+
+proc freq data=contin; tables group\*response/**JT**; title "Jonckheere-Terpstra Trend Test for Continuous Data"; run;
+
```

Comment:
Please include the test results for your simulated data using a screenshot

Reply:
> Please include the test results for your simulated data using a screenshot

Updated.